### PR TITLE
[WEB-9154] - Migrate serato/sso-auth-request to Github Actions

### PR DESCRIPTION
**Pull request by Krithik-Serato**
* Added GitHub workflow file
* Github actions now running PHPUnit tests, PHP Code sniffer, PHPStan and Security Checker
* Removed travis file

### CODE REVIEW

File: .github/workflows/automated-tests.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      - name: Install dependencies
+        uses: php-actions/composer@v6
+        with:
+          php_version: ${{ matrix.php-versions }}
+          php_extensions: gmp
+          version: 2.2
+
+      - name: Run static code analyzers
+        run: |
+          ./vendor/bin/phpcs
```
Comment (Mike-Serato) [inline .github/workflows/automated-tests.yml:45]: Can you please try adding phpstan back and pass `phpstan.neon` config file as the argument?

File: .github/workflows/automated-tests.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# This workflow is triggered every time a PR is opened, reopened, or the head branch of the pull request is updated.
```
Comment (Mike-Serato) [inline .github/workflows/automated-tests.yml:1]: Can you please update this comment as well?

**Review by AnnieCSerato — APPROVED** [review]
LGTM

**Review by Mike-Serato — APPROVED** [review]
Please squash some of those commits before merging. I added a lot of garbage commits for debugging purposes. :-D

**Pull request merged**